pkg/lib/cockpit-po-plugin.js

```diff
@@ -23,7 +23,12 @@ module.exports = class {
                     name: 'CockpitPoPlugin',
                     stage: webpack.Compilation.PROCESS_ASSETS_STAGE_ADDITIONAL,
                 },
-                () => Promise.all(glob.sync(path.resolve(srcdir, 'po/*.po')).map(f => this.buildFile(f, compilation)))
+                () => {
+                    const linguas_file = path.resolve(srcdir, "po/LINGUAS");
```

Comment:
starter-kit and friends don't have/use LINGUAS, so the glob needs to stay at least as a fallback, or the other projects need to be converted first. Right now the following readFileSync() would crash with these.

Reply:
We're quite explicit about the fact that we don't promise any API here.  We'd clearly need to add LINGUAS into the other projects before we bump the version of `pkg/lib` that they pull in.

Reply:
My point is rather that it is not really useful for starter-kit and friends: They use webpack, not make, for building all the po stuff, and webpack-watch isn't powerful enough for rebuilding on newly added files anyway. It just simply doesn't have that kind of "add dependency list" mechanism.

As discussed on IRC: I think it'd be fine to add LINGUAS to dependencies *if* it exists (i.e. for cockpit.git), and otherwise keep the globbing as-is.